docs/genai/howto/build-from-source.md

```diff
@@ -86,7 +86,7 @@ Build from source and copy the include and libraries into `ORT_HOME`
 
 On Windows
 
-```cmd
+```bash
```

Comment:
Does it matter? I think the following commands are Windows bat commands, not bash commands.